run0/src/container/mod.rs

```diff
@@ -14,19 +15,6 @@ mod namespaces;
 const OCI_RUNTIME_SPEC_FILE: &str = "config.json";
 const OCI_RUNTIME_SPEC_ROOTFS: &str = "rootfs";
 
-/// Containers related errors
-#[derive(Debug)]
-pub enum Error {
-    OCISpecificationLoad(oci_spec::OciSpecError),
-    OCIInvalidNamespace(oci_spec::runtime::LinuxNamespaceType),
-    ContainerSpawnCommand(unshare::Error),
-    ContainerWaitCommand(std::io::Error),
-    ContainerExit(i32),
-    Unmount(std::io::Error),
-}
-
```

Comment:
I'd prefer if we keep those errors in the container module, as they're container related.

The cli module would define its own errors as well, that would typically take a container error as an input:

```rust
/// CLI errors
#[derive(Debug)]
pub enum Error {
    ContainerRunError(crate::container::Error),
}
```